environment/src/main/java/com/sequenceiq/environment/environment/dto/EnvironmentBackup.java

```diff
@@ -62,4 +62,15 @@ public BackupCloudwatchParams getCloudwatch() {
     public void setCloudwatch(BackupCloudwatchParams cloudwatch) {
         this.cloudwatch = cloudwatch;
     }
+
+    @Override
+    public String toString() {
+        return "EnvironmentBackup{" +
+                "storageLocation='" + storageLocation + '\'' +
+                ", s3=" + s3 +
+                ", adlsGen2=" + adlsGen2 +
+                ", gcs=" + gcs +
+                ", cloudwatch=" + cloudwatch +
```

Comment:
Nit: The output of `BackupCloudwatchParams.toString()` mentions `CloudwatchParams` instead of the real classname `BackupCloudwatchParams`.